netlify.toml

```diff
@@ -0,0 +1,17 @@
+[build]
+base = "site/"
+publish = "site/public/"
+command = "hugo --gc"
+
+[build.environment]
+HUGO_VERSION = "0.65.1"
+
+[context.production.environment]
+HUGO_ENV = "production"
+HUGO_ENABLEGITINFO = "true"
+
```

Comment:
Wouldn't it make sense to add this already?
```
HUGO_BASEURL = "https://krew.sigs.k8s.io/"
```

Reply:
it's already in config.yaml, so I don't think it's needed.

Reply:
I see! makes sense